demo/top_down_video_demo_full_frame_without_det_gpuaccel.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import warnings
+from argparse import ArgumentParser
+
+import cv2
+import mmcv
+import numpy as np
+import torch
+from torchvision.transforms import functional as F
+
+from mmpose.apis import init_pose_model, vis_pose_result
+from mmpose.apis.inference import _box2cs
+from mmpose.datasets import DatasetInfo
+
+try:
+    import ffmpegcv
+except ImportError:
+    raise ImportError(
+        'Please install ffmpegcv with:\n\n    pip install ffmpegcv')
+
+
+def prefetch_img_metas(cfg, ori_wh):
+    w, h = ori_wh
+    bbox = np.array([0, 0, w, h])
+    center, scale = _box2cs(cfg, bbox)
+    scale /= 1.25  # never expand bbox area
+    dataset_info = cfg.data['test'].get('dataset_info', None)
+    assert dataset_info, 'Please set `dataset_info` in the config.'
+    img_metas = {
+        'img_or_path':
+        None,
+        'img':
+        None,
+        'image_file':
+        '',
+        'center':
+        center,
+        'scale':
+        scale,
+        'bbox_score':
+        1,
+        'bbox_id':
+        0,
+        'dataset':
+        dataset_info.dataset_name,
+        'joints_3d':
+        np.zeros((cfg.data_cfg.num_joints, 3), dtype=np.float32),
+        'joints_3d_visible':
+        np.zeros((cfg.data_cfg.num_joints, 3), dtype=np.float32),
+        'rotation':
+        0,
+        'ann_info': {
+            'image_size': np.array(cfg.data_cfg['image_size']),
+            'num_joints': cfg.data_cfg['num_joints'],
+            'flip_pairs': dataset_info.flip_pairs
+        }
+    }
+    for pipeline in cfg.test_pipeline[1:]:
+        if pipeline['type'] == 'NormalizeTensor':
+            img_metas['img_norm_cfg'] = {
+                'mean': np.array(pipeline['mean']) * 255.0,
+                'std': np.array(pipeline['std']) * 255.0
+            }
+            break
+    else:
+        raise Exception('NormalizeTensor is not found.')
+
+    return img_metas
+
+
+def process_img(frame_resize, img_metas, device):
+    assert frame_resize.shape[1::-1] == tuple(
+        img_metas['ann_info']['image_size'])
+    frame_cuda = torch.from_numpy(frame_resize).to(device).float()
+    frame_cuda = frame_cuda.permute(2, 0, 1)  # HWC to CHW
+    mean = torch.from_numpy(img_metas['img_norm_cfg']['mean']).to(device)
+    std = torch.from_numpy(img_metas['img_norm_cfg']['std']).to(device)
+    frame_cuda = F.normalize(frame_cuda, mean=mean, std=std, inplace=True)
+    frame_cuda = frame_cuda[None, :, :, :]  # NCHW
+    data = {'img': frame_cuda, 'img_metas': [img_metas]}
+    return data
+
+
+def main():
+    """Visualize the demo video with GPU acceleration.
+
+    Using mmdet to detect the human.
```

Comment:
Seems like we don't need `mmdet` here to detect the human.

Reply:
I copy that from original `top_down_video_demo_full_frame_without_det.py`

Reply:
OK, I see. Have you tested this demo script with some config? Does the visualization result look fine?

Reply:
This code has the same API with the original `top_down_video_demo_full_frame_without_det.py`. I've tested this code with the some config of mine and got the identical results. I hope this code will be general to other configs.